assets/wizards/seo/views/separator/index.js

```diff
@@ -1,91 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { Component, Fragment } from '@wordpress/element';
-import { decodeEntities } from '@wordpress/html-entities';
-import { __ } from '@wordpress/i18n';
-
-/**
- * Internal dependencies
- */
-import {
-	Button,
-	ButtonGroup,
-	PreviewBox,
-	SectionHeader,
-	withWizardScreen,
-} from '../../../../components/src';
-
-const SEPARATORS = {
-	'sc-dash': '-',
-	'sc-ndash': '&ndash;',
-	'sc-mdash': '&mdash;',
-	'sc-bull': '&bull;',
-	'sc-star': '*',
-	'sc-pipe': '|',
-};
-
-/**
- * SEO Separator screen.
- */
-class Separator extends Component {
-	/**
-	 * Generate sample page title using selected separator.
-	 */
-	exampleTitle = titleSeparator => {
-		const separator = SEPARATORS[ titleSeparator ];
-		const blogName =
-			window && window.newspack_urls
-				? window.newspack_urls.bloginfo.name
-				: __( 'Site Name', 'newspack' );
-		return `${ __( 'Homepage', 'newspack' ) } ${ separator } ${ blogName }`;
-	};
-
-	/**
-	 * Render.
-	 */
-	render() {
-		const { data, onChange } = this.props;
-		const { titleSeparator } = data;
-		return (
-			<Fragment>
-				<SectionHeader
-					title={ __( 'Title separator', 'newspack' ) }
-					description={ () => (
-						<>
-							{ __( 'Choose the symbol to use as your title separator', 'newspack' ) }
```

Comment:
This part was not moved to the new view – is this intentional?

Reply:
Yes, I didn't see a point in keeping this one. Most publishers use the default setting and never change this.